packages/paste-core/components/keyboard-key/src/KeyboardKey.tsx

```diff
@@ -59,11 +58,11 @@ export interface KeyboardKeyProps extends HTMLPasteProps<"div"> {
    * @type string
    * @memberof KeyboardKeyProps
    */
-  keyText?: string;
+  keyEvent?: string;
```

Comment:
suggestion (non-blocking): maybe add to the description comment here that the docs have a chart of usable keyEvents? or a link to the map that you were using at one point?

slash would it make sense to limit the type of `keyEvent` from `string` to an actual list of usable key codes? I'm guessing no for scalability, but wanted to ask

Reply:
A really great suggestion. Pushing up the comment update. We can use the `event.which` which is the code number. It's something I feel developers would be more used to but it changes based on browser as mentioned in the link to the table. It is also deprecated so browsers at some point in the future may stop returning that field.